pkg/kv/kvserver/gc/gc_random_test.go

```diff
@@ -36,10 +36,11 @@ import (
 // running randomized GC tests as well as benchmarking new code vs preserved
 // legacy GC.
 type randomRunGCTestSpec struct {
-	ds           distSpec
-	now          hlc.Timestamp
-	ttlSec       int32
-	intentAgeSec int32
+	ds             distSpec
+	now            hlc.Timestamp
+	ttlSec         int32
+	intentAgeSec   int32
+	clearRangeKeys int64 // Set to 0 for test default value or -1 to disable.
```

Comment:
nit: probably not worth a CI roundtrip, but using the same name `clearRangeMinKeys` as the actual parameter would be clearer.